using-kontena/nodes.md

```diff
@@ -17,12 +17,29 @@ In this chapter, we'll discover how to manage, operate and monitor Kontena Nodes
 
 Please see the [Add Kontena Nodes](install-nodes/README.md) documentation to learn more.
 
+## Create Kontena Nodes
+
+Create a new Kontena Node for manual provisioning:
```

Comment:
> Create Kontena Node stuff should be moved to install-nodes/README.md and/or one of its sub documents

So the `using-kontena/nodes.md` should not have any mention of `kontena node create`?

Writing up the detailed docs in `install-nodes/README.md` is still WIP, I can move this to that branch.